src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -170,7 +170,7 @@ See Cloudflare's documentation for [working with environment variables](https://
 ```js
 // pages/[id].json.js
 
-export function GET({ params }) {
+export function get({ params }) {
```

Comment:
Huh this looks wrong again. Something to tackle next week 🤷

Reply:
Have created https://github.com/withastro/astro/pull/8429 to fix this at the source.